docs/whatsnew/7.1.rst

```diff
@@ -45,6 +45,52 @@ is now supported by the Unified File Read/Write Interface.
 Note that this format is compatible with ``Table`` objects with integer, float,
 and/or string data types, but not ``QTable`` objects or other data types.
 
+.. _whatsnew-7.1-cosmology:
+
+Cosmology
+=========
+
+
+The `astropy.cosmology.traits` module hosts various parts of cosmologies, such as the
+:class:`~astropy.cosmology.traits.ScaleFactor` or
+:class:`~astropy.cosmology.traits.TemperatureCMB`. These traits can be used to more
+easily construct custom cosmologies by combining different components.
+
+As a simple example, the :class:`~astropy.cosmology.traits.TemperatureCMB` trait
+provides the ``Tcmb0`` property and
+:meth:`~astropy.cosmology.traits.TemperatureCMB.Tcmb` method for computing the
+cosmological CMB temperature at specified redshifts. By using this trait, you can add
+temperature-related  functionality to your custom cosmology class without having to
+implement it from scratch.
+
+Here is an example of how to use the :class:`~astropy.cosmology.traits.ScaleFactor` and
+:class:`~astropy.cosmology.traits.TemperatureCMB` traits in a custom cosmology class:
+
+>>> import astropy.units as u
+>>> from astropy.cosmology.traits import ScaleFactor, TemperatureCMB
+>>> from astropy.cosmology import Cosmology
+
```

Comment:
Are you sure these empty lines look good in HTML?

Reply:
It does look better without.

Reply:
It does look better to connect here at the import, but not lower when making ``cosmo``.